packages/@lwc/shared/src/api-version.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
```

Comment:
```suggestion
 * Copyright (c) 2023, salesforce.com, inc.
```

Reply:
Guidance from legal is any new file gets this year in the copyright header. Any substantial changes to a file should also result an update to the copyright year (although that's harder to remember).

Reply:
We should probably just do a search-and-replace across our whole codebase then. And do it on January 1st. Otherwise it's too easy to forget. 😛

Reply:
I just read a whole article on this and 🤯